packages/core/src/network.js

```diff
@@ -354,7 +354,7 @@ async function sendResponseResource(network, request, session) {
   let send = (method, params) => network.send(session, method, params);
 
   try {
-    let resource = network.intercept.getResource(url);
+    let resource = network.intercept.getResource(url, network.currentWidth);
```

Comment:
apart from `network.currentWidth` is there any way to get any width this looks little weird.

Reply:
we can use cdp to get current window width but that will be slower operation as compared to this.

Reply:
I have commented to make it better